assets/js/components/Dashboard/PlacesActionBar.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import { Button } from "react-bootstrap";
+import {
+  ListCheck,
+  XCircleFill,
+  EyeFill,
+  EyeSlashFill,
+  Icon,
+} from "react-bootstrap-icons";
+import { Place } from "../../models/place";
+
+interface Props {
+  places: Place[];
+  hasScreenlessPlaces: boolean;
+  showScreenlessPlaces: boolean;
+  onClickResetFilters: () => void;
+  onClickToggleScreenlessPlaces: () => void;
+}
+
+const getPlaceCount = (places: Place[]) => {
+  return places.length;
+};
+
+const getScreenCount = (places: Place[]) => {
+  // Get all screens within the filtered places.
+  // TODO: Should we also limit the count to only include screens of the filtered screen type, when that filter is selected?
```

Comment:
I don't think we've addressed this particular question before, but a similar question was brought up for alerts. For alerts, someone asked if you are filtering by a specific alert should only screens showing that alert be displayed. The answer to that was no, show all screens at all places showing that alert on at least one screen.